CNV-30835: must-gather instancetypes flag (#62954)
openshift/openshift-docs: modules/virt-using-virt-must-gather.adoc

* Namespaces that contain virtual machines
* Basic virtual machine definitions

> InstanceTypes information is not currently collected by default; you can, however, run a command to optionally collect it.

Reviewer: In my opinion, this is redundant information from the other "must-gather tool options" section. So maybe it can be removed? 

But if you decide to keep this information here then below are my suggestions:

```suggestion
InstanceTypes information is not currently collected by default. Optionally, you can run a command to collect it.
```

Reviewer: Also, it might be helpful to mention which command.
For eg: "Optionally, you can run the "xyz" command to....."

Reviewer 2: I agree with @xenolinux. While the [Usage and examples](https://62954--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-collecting-virt-data.html#usage-and-examples_virt-collecting-virt-data) section mentions the command, it might be worth adding it to this module. For example:
```suggestion
InstanceTypes information is not currently collected by default. To collect this information, run the following command:
```

Also, should this be an admonition? For example, a note?

Author: The commands for the other must-gather options are at the end of the module, so I'd like to leave the one for instance types there, too
